### PR TITLE
Story/vspc 12

### PR DESCRIPTION
**Pull request by namrathaov**
Added dashboard link at /staff/dashboard
Dashboard displays latest 5 spaces and modules.

### CODE REVIEW

**Comment by jdamerow** [comment]
test this please

**Comment by jdamerow** [comment]
test this please

File: vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboardlist.jsp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+<h3>Spaces</h3>
+
+<div style="padding-bottom: 20px;">
+	This virtual exhibition contains the following spaces.
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboardlist.jsp]: Should say: "Recently created spaces:"

File: vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboardlist.jsp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	  (Created on <span class="date">${space.creationDate}</span> by ${space.createdBy})
+    </li>
+  </c:forEach>
+</ul>
+<br/>
+
+<h3>Modules</h3>
+
+<div style="padding-bottom: 20px;">
+	This virtual exhibition contains the following modules.
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboardlist.jsp]: Should say: "Recently created modules:"

**Comment by namrathaov** [comment]
Added minor changes in labels for dashboard page.

**Comment by namrathaov** [comment]
"After logging in, there should be a link to the dashboard from the exhibition layout."
VSPC-11 is a similar task to add a menu link to the space list. Should there be a dashboard link too in the home page?

**Comment by jdamerow** [comment]
Ya, VSPC-11 was meant to be there before this was done. Now that this is almost done, we can scrap VSPC-11.

**Comment by jdamerow** [comment]
link still missing

**Comment by namrathaov** [comment]
Added Dashboard link for staff user.

**Pull request merged**